### PR TITLE
[e2e-move-tests] remove aptos crate (cli tool) dependency

### PR DESCRIPTION
**Pull request by igor-aptos**
e2e-move-tests were depending on aptos crate, just to use two utility structs. With that it basically required everything to be built. 

moving these to aptos-types (let me know if you have a better preference), reduces number of dependencies from ~1400 to ~1000, and significantly reducing build time. 
Also when modifying some code, and re-running, much less things need to be build, and the very-time-consuming "aptos" crate is not built any more

### CODE REVIEW

**Review by msmouse — APPROVED** [review]

**Comment by igor-aptos** [comment]
ping, need a second review :)

**Review by vgao1996 — APPROVED** [review]

**Review by wrwg — APPROVED** [review]
Thanks for doing this!

**Review by banool — APPROVED** [review]
Looks good from the CLI perspective

**Pull request merged**
